app/views/main.scala.html

```diff
@@ -23,6 +24,10 @@
 		<meta property="og:description" content="@description" />
 	}
 
+	@if(canonicalLink.isDefined) {
```

Comment:
You may simply`.map` here: `@canonicalLink.map { link => ... }`

Reply:
I'm using the style `description.isDefined` uses right above, they should match IMO

Reply:
I just opted to map over both as I think your suggestion makes it clearer @LATaylor-guardian